packages/amazon-ssm-agent/amazon-ssm-agent.spec

```diff
@@ -65,7 +65,7 @@ Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-plugin-bin)
 %{summary}.
 
 %prep
-%autosetup -n %{gorepo}-%{version} -p0001
+%autosetup -n %{gorepo}-%{version} -p1
```

Comment:
This change doesn't relate to your PR. You can cherry pick the commit into another PR, along with the `socat` update.

Reply:
Ack, I included these fix commits in this PR as the issue was brought up here, but I can create a new one for them and remove them from this PR